ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -274,6 +274,36 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure Internet Sharing Is Disabled
+  platforms: macOS
+  platform: darwin
+  description: |
+    Apple Events is a technology that allows one program to communicate
+    with other programs. Remote Apple Events allows a program on one computer
+    to communicate with a program on a different computer.
+    Disabling Remote Apple Events mitigates the risk of an
+    unauthorized program gaining access to the system
+  resolution: |
+    Graphical Method:
+    1. Open System Settings
+    2. Select General
+    3. Select Sharing
+    4. Set Internet Sharing to disabled
+  query: |
+    SELECT 1 WHERE NOT EXISTS (
+      SELECT * FROM plist WHERE
```

Comment:
Does this query need `SELECT *` ?
Would `SELECT 1` not work?

Reply:
Both would work. The result is the same because of the outer `SELECT 1`. (`SELECT 1` "means" return `1` if there are one or more results.)

Reply:
This was a pattern I saw in one query, though it could possibly be more efficient to use `SELECT 1` there. It does work though